### ISSUE TITLE
Quantized version of Sigmoid doesn't have _get_name method

### ISSUE THREAD

**Issue by L-ED** [pytorch/pytorch#85968]
### 🐛 Describe the bug

Hello, i'm interested in fx quantization of pytorch model. In my project i need to register model by its class name, for example:
I define module class from GraphModule by calling node.target. For quantized version of ConvBnReLU output would be QuantizedConvBnReLU.

Despite quantized Sigmoid my method works for any quantized module because of “Quantized” prefix in module class name.

I checked realization of quantized activations and Sigmoid doesn’t contain method _get_name like any other realization of quantized activation.

### Versions

wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py

cc @jerryzh168 @jianyuh @raghuramank100 @jamesr66a @vkuzo @jgong5 @Xia-Weiwen @leslie-fang-intel

**Comment by jerryzh168** [comment]
thanks for reporting the issue. could you check if we are using torch.nn.quantized.Sigmoid or torch.nn.Sigmoid here? my guess is that we are using torch.nn.Sigmoid right now

**Comment by jerryzh168** [comment]
sorry, in the quantized model, which one did you see?

**Comment by andrewor14** [comment]
Hi @L-ED, is this still an issue? Did you find a workaround?

**Comment by andrewor14** [comment]
Hi @L-ED, I'm closing this for now since relying on module name having "Quantized" isn't a robust way to check whether the module is actually quantized. If you don't see quantized sigmoid in the model then that's a different issue, in which case feel free to open a separate issue.
